jdaviz/configs/imviz/helper.py

```diff
@@ -105,6 +107,108 @@ def load_data(self, data, parser_reference=None, **kwargs):
             self.app.load_data(
                 data, parser_reference=parser_reference, **kwargs)
 
+    def link_data(self, link_type='pixels', wcs_fallback_scheme='pixels', wcs_use_affine=True,
+                  error_on_fail=False):
+        """(Re)link loaded data with the desired link type.
+        All existing links will be replaced.
+
+        .. warning::
+
+            Any markers added would be removed. You can add back the markers
+            manually using :meth:`add_markers`. During the markers removal,
+            pan/zoom will also reset.
+
+        Parameters
+        ----------
+        link_type : {'pixels', 'wcs'}
+            Choose to link by pixels or WCS.
+
+        wcs_fallback_scheme : {None, 'pixels'}
+            If WCS linking failed, choose to fall back to linking by pixels or not at all.
+            This is only used when ``link_type='wcs'``.
+            Choosing `None` may result in some Imviz functionality not working properly.
+
+        wcs_use_affine : bool
+            Use affine transform to approximate the WCS alignment, ignoring distortion.
```

Comment:
This doesn't ignore distortion as such because two images that have distortions and are offset might have their difference well approximated by an affine transform. The affine transform is not approximating the WCSes but the difference between the images.

Reply:
Yes in that case the approximation would fail - you can set the tolerance to whatever you need so to some extent that tolerance is what matters not whether distortions are present or not. You can have situations without distortions that are not well approximated too.

Reply:
@astrofrog , please suggest new concise wordings that are more accurate then. I am not sure what to write and I don't feel like turning the docstring into an essay about WCS.

Reply:
"Use an affine transform to represent the offset between images

Reply:
Done. Thanks for the suggestion!